importexport/musicxml/exportxml.cpp

```diff
@@ -5987,9 +5987,10 @@ void ExportMusicXml::print(const Measure* const m, const int partNr, const int f
 
                         if (mpc.systemStart && !mpc.pageStart) {
                               // see System::layout2() for the factor 2 * score()->spatium()
+                              const Measure* prevSystem = mpc.prevMeasure->isMMRest() ? mpc.prevMeasure->mmRestFirst() : mpc.prevMeasure;
```

Comment:
@rettinghaus: As Mu3 doesn't have `coveringMMRestOrThis()`, I'm using your earlier approach here, is that OK?
Interestingly this PR here passes the mtests, but your's doe not?!ß

Reply:
Does it work? My tests didn't, that's why I changed the approach. I'd be grateful if you could help me with that test.

Reply:
Well, the tests pass here